src/harness/fake_sas.py

```diff
@@ -177,14 +177,15 @@ def _GetMissingParamResponse(self):
     return {'responseCode': MISSING_PARAM}
 
   def InjectEscZone(self, request, ssl_cert=None, ssl_key=None)
-    zoneData = req['zoneDate']
-    if(zoneData['usage'] == 'EXCLUSION_ZONE'):
-      idAttributes = zoneData['id'].split('/')
-      zoneId = ''; 
-     for zoneIdPart in idAttributes[4 : len(idAttributes)]:
-       zoneId+=zoneIdPart
-     return {'zone_id' : zoneId}
-		
+    zone_data = request['zoneData']
+    if(zone_data['usage'] == 'EXCLUSION_ZONE'):
```

Comment:
This actually raises a good point: ZoneData isn't originally meant to be used for ESC so it's not clear what the value of "usage" should be in this case. Maybe allowing the field to be empty is better than using something that may be misleading?

Whatever we decide, we should make sure we write it down in the API doc.